examples/hosts/app-integration/external-data/src/mock-service/externalData.ts

```diff
@@ -54,37 +27,42 @@ export interface IExternalDataSourceEvents extends IEvent {
 
 /**
  * Class to let us fake having an external data source and abstract the particulars of its implementation.
+ *
+ * @remarks
+ *
  * In a more-real scenario, maybe this is communicating with some server via RESTful APIs.
+ *
  * It's an event emitter just so we can render a reasonable debug view on it for demo purposes - in more-realistic
  * cases we would expect to learn about data updates through webhooks or similar.
- * TODO: Implement a debug control to simulate data changing remotely, webhook, etc.
+ *
+ * @privateRemarks
+ *
  * TODO: Consider adding a fake delay to the async calls to give us a better approximation of expected experience.
- * TODO: This will probably want to move to a standalone Express server or something eventually, esp. when working
- * on the server-side bot approach.  But using localStorage is probably good enough for the broadcast signal portion.
  */
 export class ExternalDataSource extends TypedEventEmitter<IExternalDataSourceEvents> {
+    private data: string;
```

Comment:
Now that the data source is managed by the background service, we no longer need to use `window` storage to make the data cross-tab accessible.